include/ceed-impl.h

```diff
@@ -131,11 +131,34 @@ struct CeedQFunction_private {
   CeedInt numinputfields, numoutputfields;
   int (*function)(void*, CeedInt, const CeedScalar *const*, CeedScalar *const*);
   const char *focca;
+  bool fortranstatus;
   void *ctx;      /* user context for function */
   size_t ctxsize; /* size of user context; may be used to copy to a device */
   void *data;     /* backend data */
 };
 
+/// Struct to handle the context data to use the Fortran QFunction stub
+/// @ingroup CeedQFunction
+typedef struct {
+  CeedScalar *innerctx;
```

Comment:
I set the Fortran context to only be arrays of scalars. I'm not entirely sure Fortran77 does user defined structs in the same way C90 does, and this should still offer the desired functionality, as ints and bools can be saved into scalars and then recovered.

Reply:
F77 doesn't have structs. F90 does, but they repeatedly botched the interface specification such that it's only practical without vendor-specific extensions after F2008+TS29113 (or F2018).